stdlib/3/multiprocessing/synchronize.pyi

```diff
@@ -0,0 +1,53 @@
+from typing import Callable, ContextManager, Optional, Union
+
+from multiprocessing.context import BaseContext
+import threading
+
+class Barrier(threading.Barrier):
+    def __init__(self,
+                 parties: int,
+                 action: Optional[Callable] = ...,
+                 timeout: Optional[float] = ...) -> None: ...
+
+class BoundedSemaphore(Semaphore):
+    def __init__(self, value: int = ..., *, ctx: BaseContext) -> None: ...
+
+_LockLike = Union[Lock, RLock]
+
+class Condition(ContextManager[bool]):
```

Comment:
Some of these classes also apparently have `acquire` and `release` methods that are set at runtime in `_make_methods`.